docs/en/plugins/README.md

```diff
@@ -0,0 +1,413 @@
+Loomio’s recently released a plugin architecture, which will allow you quickly and easily add functionality to Loomio on your own instance. There’s some good docs in (plugin README), but I thought it might be helpful to step through how we make plugins here, step by step
+
```

Comment:
Loomio’s recently released ~~a~~ plugin architecture, which will allow you **to** quickly and easily add functionality to Loomio on your own instance. There~~'s~~ **are** some good docs in (plugin README), but I thought it might be helpful to step through how we make plugins here~~, step by step~~**.**